### PR TITLE
Added an part 1 of an example for transferring HBAR to a contract

### PR DESCRIPTION
**Pull request by ed-marquez**
**Description**:

**Checklist**
- [x] Documented (Code comments, article in hedera.com/blog)
- [x] Tested

### CODE REVIEW

File: examples/transfer-hbar2contracts-sdk/transferHbar2Contract_sdk.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+async function main() {
+	// Create additional accounts needed
+	const initialBalance = 100;
+	const [accStatus, aliceId] = await accountCreatorFcn(aliceKey, initialBalance);
+	console.log(
+		`\n- Created Alice's account with initial balance of ${initialBalance} hbar: ${accStatus}`
+	);
+
+	// Import the compiled contract bytecode
+	const contractBytecode = fs.readFileSync("transferHbar2Contract_sdk_sol_hbar2Contract.bin");
```
Comment (Nana-EC) [inline examples/transfer-hbar2contracts-sdk/transferHbar2Contract_sdk.js:31]: nit: folder already gives context
```suggestion
	const contractBytecode = fs.readFileSync("hbar2Contract.bin");
```

File: examples/transfer-hbar2contracts-sdk/transferHbar2Contract_sdk.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+const operatorKey = PrivateKey.fromString(process.env.OPERATOR_PVKEY);
+const aliceKey = PrivateKey.generateED25519();
+const client = Client.forTestnet().setOperator(operatorId, operatorKey);
+
+async function main() {
+	// Create additional accounts needed
+	const initialBalance = 100;
+	const [accStatus, aliceId] = await accountCreatorFcn(aliceKey, initialBalance);
+	console.log(
+		`\n- Created Alice's account with initial balance of ${initialBalance} hbar: ${accStatus}`
```
Comment (Nana-EC) [inline examples/transfer-hbar2contracts-sdk/transferHbar2Contract_sdk.js:27]:
```suggestion
		`\n- Created Alice's account '${aliceId}' with initial balance of ${initialBalance} hbar: ${accStatus}`
```

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Thanks for the example.
Debating whether this repo or sdk-js is the best location.
For now it's okay.

Also may want to consider the following
- making it a complete npm package that's easy to run
- adding a README

**Comment by ed-marquez** [comment]
Addressed the changes suggested and rebased the branched to conform to DCO checks.

**Review by SimiHunjan — APPROVED** [review]

**Comment by ed-marquez** [comment]
Closing this merge requests for now. I just added another examples for a 2nd part of the article. Will create a new requests that includes both examples

**Pull request closed without merging**